src/views/DidDowngrade/DidDowngrade.tsx

```diff
@@ -145,16 +156,27 @@ export function DidDowngrade({ identity }: Props): JSX.Element | null {
 
       <IdentitySlide identity={identity} options={false} />
 
-      <p className={styles.costs}>
-        {t('view_DidDowngrade_total')}
-        <KiltAmount amount={total} type="costs" smallDecimals />
-      </p>
-      <p className={styles.details}>
-        {t('view_DidDowngrade_deposit')}
-        {asKiltCoins(deposit, 'costs')} <KiltCurrency />
-        {t('view_DidDowngrade_fee')}
-        {asKiltCoins(fee, 'costs')} <KiltCurrency />
-      </p>
+      {isDepositOwner && (
+        <Fragment>
+          <p className={styles.costs}>
+            {t('view_DidDowngrade_total')}
+            <KiltAmount amount={total} type="costs" smallDecimals />
+          </p>
+          <p className={styles.details}>
+            {t('view_DidDowngrade_deposit')}
+            {asKiltCoins(deposit, 'costs')} <KiltCurrency />
```

Comment:
why is this listed as costs? you get the deposit back on downgrading, if you are the owner

Reply:
Good point. I updated it here and in `W3NRemove`